experimenter/experimenter/nimbus_ui_new/templates/changelog/overview.html

```diff
@@ -1,42 +1,90 @@
-{% extends "layout/with_sidebar.html" %}
+{% extends "common/with_sidebar.html" %}
 
 {% block title %}
-  History
+History
 {% endblock %}
 
 
 {% block sidebar %}
-  {% include "common/experiment_details_sidebar.html" with experiment=experiment %}
+{% include "common/experiment_details_sidebar.html" with experiment=experiment %}
 {% endblock %}
 
 {% block main_content %}
-  <div class="p-5 sm:p-8 space-y-4">
-    <div class="mb-6 space-y-1">
-      <h1 class="text-2xl sm:text-3xl text-gray-800 font-medium">Changes Overview</h1>
-      <p class="text-sm sm:text-base text-gray-500">{{ experiment.slug }}</p>
+<div class="container-fluid">
+  {% for changelog in experiment.get_changelogs_by_date %}
+  <div class="row">
+    <div class="col-1 text-center py-1" style="z-index: 1;">
+      <span class="badge rounded-pill text-bg-light text-primary shadow">
+        <i class="fa-regular fa-calendar fs-5"></i>
+      </span>
+      <div class="d-flex justify-content-center" style="height: 100%;">
+        <div class="vr" style="width:3px;background-color: #ccc;"></div>
+      </div>
     </div>
-    <div class="flex justify-center max-w-screen-2xl w-full h-full  flex-col items-center px-2 sm:px-4">
-      <ol class="border-l-2 border-gray-800 w-full">
-        {% for changelog in experiment.get_changelogs_by_date  %}
-          <li class="mb-8">
-            <div class="flex-start flex items-center">
-              <div class="-ml-[16px] -mt-2 mr-3 flex  items-center justify-center rounded-full p-2 floating bg-white">
-                <i class="fa-regular  fa-calendar  text-azure"></i>
-              </div>
-              <h4 class="-mt-2 text-lg sm:text-xl font-semibold text-gray-600">{{ changelog.date }}</h4>
+    <div class="col-11">
+      <span class="fs-5 fw-semibold">
+        {{ changelog.date }}</span>
```

Comment:
Why is this formatted like this?

Can we get an auto-formatter pass for django templates?

Reply:
Huzzah! https://github.com/mozilla/experimenter/issues/10656